server/test/data/database.js

```diff
@@ -0,0 +1,182 @@
+var assert = require('assert');
+
+const db = require('../../data/database.js');
+describe('#connect_to_db', function () {
+  it('should success with no error', async function () {
+    db.connect_to_db();
+  });
+});
+
+var user1_id;
+describe('database.js: user-related tests', function () {
```

Comment:
This test looks good! Maybe we can file an issue in the GitHub issue if this will increase the user id. 
I am wondering what will happen if the test is automatically run with every commit. Will it increase the user_ID by 5 for every commit?